admiral.Rproj

```diff
@@ -19,5 +19,5 @@ LineEndingConversion: Posix
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
-SpellingDictionary: en_US
 
+SpellingDictionary: en_US
```

Comment:
why is this file changed?

Reply:
That's strange and I'm not sure tbh. I will revert it back.